bonfire/resources/reservation-template.yaml

```diff
@@ -13,6 +13,7 @@ objects:
   spec:
     duration: ${DURATION}
     requester: ${REQUESTER}
+    pool-type: ${POOLTYPE}
```

Comment:
I suspect we are going to ask that the key name here be renamed to `poolType` over in https://github.com/RedHatInsights/ephemeral-namespace-operator/pull/65/files#diff-14d05dc348d738e3ff5d3d6a0ca563b6616adb212c02c12b6628be117d81e081R30 -- so you'll probably want to update it here too :)

Reply:
Good point. Going to shorten it to just "pool" instead.